src/components/common/UserDetails.tsx

```
@@ -104,7 +122,7 @@ const UserDetails: React.FC<UserDetailsProps> = ({ userData }) => {
 									{row.map((field) => (
 										<Field
 											label={field.label}
-											value={field.value}
+											value={processFieldValue(field, userData?.dob)}
```

ESLint

[error] Replace `field,·userData?.dob` with `⏎↹↹↹↹↹↹↹↹↹↹↹↹field,⏎↹↹↹↹↹↹↹↹↹↹↹↹userData?.dob⏎↹↹↹↹↹↹↹↹↹↹↹`

(prettier/prettier)